src/client/src/Stream.c

```diff
@@ -3245,3 +3246,46 @@ STATUS resetStream(PKinesisVideoStream pKinesisVideoStream)
     return retStatus;
 }
 
+VOID logStreamInfo(PStreamInfo pStreamInfo) {
+    PCHAR streamingType;
+
+    if (pStreamInfo == NULL) {
+        return;
+    }
+
+    switch (pStreamInfo->streamCaps.streamingType) {
+        case STREAMING_TYPE_REALTIME:
+            streamingType = (PCHAR) "REALTIME";
+            break;
+        case STREAMING_TYPE_OFFLINE:
+            streamingType = (PCHAR) "OFFLINE";
+            break;
+        case STREAMING_TYPE_NEAR_REALTIME:
+            streamingType = (PCHAR) "NEAR_REALTIME";
+            break;
+    }
+
+
+    DLOGD("Kinesis Video Stream Info:\n\t"
```

Comment:
You might want to split this into a couple of DLOGDs as it can overflow the buffer and print out only portion